resource/api/swagger.yml

```diff
@@ -208,60 +210,65 @@ paths:
       produces:
         - application/json
       responses:
-        '200':
+        "200":
           description: Domains returned. May be empty.
           schema:
             type: array
             items:
-              $ref: '#/definitions/Websites'
-        '400':
+              $ref: "#/definitions/Websites"
+        "400":
           description: Bad request.
           schema:
-            $ref: '#/definitions/Error'
-        '500':
+            $ref: "#/definitions/Error"
+        "500":
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
-            $ref: '#/definitions/Error'
+            $ref: "#/definitions/Error"
   /websiteCreator/prepare:
     put:
       operationId: websiteCreatorPrepare
+      consumes:
+        - multipart/form-data
       parameters:
-        - in: body
-          name: body
+        - in: formData
+          name: url
+          type: string
+          pattern: "^[a-z0-9]+$"
```

Comment:
what happens if you don't follow this rule ? Does the middle reject the request directly ?

Reply:
you have an error 605 : url need to follow ^[a-z0-9]+$ pattern